server/schema/submissions/createSubmission.js

```diff
@@ -0,0 +1,30 @@
+const schema = {
+  required: ['user_submission', 'output', 'submission_time'],
+  properties: {
+    user_submission: {
+      type: 'string',
+    },
+    output: {
+      type: 'string',
+    },
+    submission_time: {
+      type: 'object',
+      format: 'date-time',
```

Comment:
same thing is done in `createContestSchema`....should we change that too?

Reply:
Oh, for that mysql NOW() is not going to work, so I think it should be allowed, but the `type` should be `integer` and we can create a custom validation which checks maybe if it is a valid timestamp ?

Reply:
I guess, it is better to remove `submission_time` from this schema

Reply:
Because submission_time would not be sent in the request, otherwise it could be manipulated by anyone

Reply:
Server will itself assign a time using `new Date()` as the submission time. We wouldn't use MySQL NOW(), because for contest timings etc. we are using server date, so let's keep all the dates of the server clock

Reply:
Because it's not necessary that mysql server clock is synced with our backend server clock

Reply:
Yeah I think that is much better !